appveyor.yml

```diff
@@ -4,7 +4,6 @@ environment:
 
 install:
   - ps: Install-Product node $env:nodejs_version
-  - npm install
```

Comment:
This `npm install` is still needed to download dependencies for the root module. Without this step, `npm run bootstrap` won't work as `lerna` is not even installed.